javascript/package.json

```diff
@@ -38,7 +38,7 @@
     "typescript": "5.9.3"
   },
   "dependencies": {
-    "@cucumber/messages": ">=19.1.4 <30"
+    "@cucumber/messages": "<31"
```

Comment:
This doesn't look right.

Check renovate `rangeStrategy`.

Reply:
Looks like we have to use "or" ranges instead.

https://docs.renovatebot.com/configuration-options/#rangestrategy